cmd/chartsvc/models/chart.go

```diff
@@ -62,3 +60,10 @@ type ChartFiles struct {
 	Readme string
 	Values string
 }
+
+// ChartLatest express the latest version for a chart in a repo
+type ChartLatest struct {
```

Comment:
the name of this struct is a bit strange, maybe it's better if we call this `ResolveRepoResponse`. Also I'm wondering if we should try to follow the `apiResponse` structure (https://github.com/helm/monocular/blob/e607f09567c23a6d0401facfc77eb9df0d4aeaf6/cmd/chartsvc/handler.go#L48) to keep the API more consistent. We could use `apiListResponse` with each ID being the name of the repo and attributes for the chartName and latestVersion. WDYT?

Reply:
I am not sure if this comment still applies?